service/src/main/java/bio/terra/workspace/db/ApplicationDao.java

```diff
@@ -67,6 +67,19 @@ public class ApplicationDao {
           + "  WHERE W.application_id = A.application_id AND workspace_id = :workspace_id) AS ecount"
           + " FROM application A";
 
+  // Query for finding applications that should be enabled on the target workspace
+  private static final String WORKSPACE_CLONE_QUERY =
+      """
+      SELECT A.application_id FROM enabled_application W, application A
+      WHERE
+        W.workspace_id = :workspace_id AND
+        W.application_id = A.application_id AND
+        A.state = :operating;
+      """;
+
+  private static final RowMapper<String> WORKSPACE_CLONE_ROW_MAPPER =
```

Comment:
nit: APPLICATION_ID_ROW_MAPPER, or something else that indicates what this is mapping